### DISCUSSION TITLE
PIL requirement

### DISCUSSION THREAD

**Discussion by ElectricCandlelight** [Akascape/CTkMessagebox#4] [General]
Great work on making a good comprehensive message box that works well with CTK

Just thought I'd let you know that `pip install Pillow` is required for it to work and isn't listed on the readme and doesn't install automatically install with `pip install CTkMessagebox`

**Comment by Akascape** [comment]
@ElectricCandlelight 
Nor customtkinter has any pillow requirements ig. But it also needs PIL.Image in order to work with images.

Moreover pillow is widely installed. Will add it in the setup of next update. (Currently customtkinter is added only)

**Reply by ElectricCandlelight** [reply]
Think it's because you can use pretty much all of CTK without using images.

I get that a lot of people have pillow installed but a lot of people will use a venv with just the standard library when starting a new project.

Would be nice if this could be merged into CTK cause it's a handy library.

**Reply by Akascape** [reply]
Just updated the module. Added the pillow requirement this time.
